tests/testthat/test-infrastructure.r

```diff
@@ -59,3 +59,96 @@ test_that("use_data", {
     expect_equal(local_test_data_item_to_save, 42L)
   })
 })
+
+test_that("add_desc_package only modifies 'field' section of DESCRIPTION", {
+  # use tests/testthat/testData as testing package
+  path_to_test_desc <- file.path("testData", "DESCRIPTION")
+  old_desc <- read_dcf(path_to_test_desc)
+
+  # be sure to return the description to original state on exit
+  on.exit(write_dcf(path_to_test_desc, old_desc))
+
+  # add testPackage to the "Imports" of the testData package
+  import_text <- "testPackage (>= 1.0.0)"
+  output <- add_desc_package("testData", "Imports", import_text)
+  new_desc <- read_dcf(path_to_test_desc)
+
+  expect_true(output) # check that add_desc_package still returns boolean
+  expect_equal(old_desc, new_desc[which(names(new_desc) != "Imports")]) # make sure only Imports section is modified
+  expect_equal(new_desc[["Imports"]], import_text) # make sure Import section is properly modified
+})
+
+test_that("add_desc_package is idempotent", {
+  # use tests/testthat/testData as testing package
+  path_to_test_desc <- file.path("testData", "DESCRIPTION")
+  old_desc <- read_dcf(path_to_test_desc)
+
+  # be sure to return the description to original state on exit
+  on.exit(write_dcf(path_to_test_desc, old_desc))
+
+  # add testPackage to the Imports: of the testData package;
+  # do it twice and check that the description file is the same as in
+  # the second time.
+  import_text <- "testPackage (>= 1.0.0)"
+  descriptions <- lapply(c(1,2), function(call_repetition) {
```

Comment:
Can you eliminate the `lapply()`? I think copying and pasting is fine if you only have two copies of basically the same code.